tests/unity/wrap_test/src/fns/fns.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/*
+	DO NOT FORMAT THIS FILE!
+
+	It contains function declarations that is used to validate the test wrapper
+	scripts (func_name_list.py and header_parse.py) which renames the functions.
+	The regular expression has had several bugs in the past, and the formating
+	of the functions here are used to validate that	the fixes still work when
+	modifying the regex.
+
+	List of fixes this file tries to reproduce:
```

Comment:
```suggestion
	List of earlier bugs this file tries to reproduce:
```

Reply:
I don't hope it reproduces the bug 😆 

How about:
```suggestion
	The examples in this file ensures that the test wrapper scripts can handle the situations described in following list of issues:
```